docs/_pages/releases.md

```diff
@@ -151,6 +151,7 @@ sidebar:
 * The `IEquivalencyValidationContext` has undergone significant refactorings where its properties have been moved into the `INode` hierarchy and the two `Reason` and `Tracer` classes - [#1379](https://github.com/fluentassertions/fluentassertions/pull/1379)
 * The `IMemberMatchingRule`, `IMemberSelectionRule` and `IOrderingRule` have been changed to replace their dependency on `SelectedMemberInfo` to `INode` and its derivatives `IMember` - [#1379](https://github.com/fluentassertions/fluentassertions/pull/1379)
 * The `SelectedMemberInfo` class has been removed, since it main user, `IMemberInfo` has been flattened - [#1379](https://github.com/fluentassertions/fluentassertions/pull/1379)
+* `IMemberInfo.SelectedMemberInfo`, `SelectedMemberPath` and `SelectedMemberDescription` have been replaced by `Name`, `Type`, `DeclaringType` and `Path` - [#1379](https://github.com/fluentassertions/fluentassertions/pull/1379)
```

Comment:
I'm not sure I understand what properties have been replaced with what.
E.g. the three old properties have been replaced by four new.

Reply:
Rephrased it. Hopefully it's better now.